iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/LoadAsymmetricalImpl.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Load;
+import com.powsybl.iidm.network.extensions.LoadAsymmetrical;
+
+/**
+ * @author Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class LoadAsymmetricalImpl extends AbstractExtension<Load> implements LoadAsymmetrical {
+
+    // This class is used as an extension of a "classical" balanced direct load
+    // we store here the deltas of power that will build the unblalanced loads. The reference is the positive sequence load stored in "Load"
+    private double deltaPa;
+    private double deltaQa;
+    private double deltaPb;
+    private double deltaQb;
+    private double deltaPc;
+    private double deltaQc;
+
+    public LoadAsymmetricalImpl(Load load, double deltaPa, double deltaQa, double deltaPb, double deltaQb, double deltaPc, double deltaQc) {
+        super(load);
+        this.deltaPa = deltaPa;
+        this.deltaPb = deltaPb;
+        this.deltaPc = deltaPc;
+        this.deltaQa = deltaQa;
+        this.deltaQb = deltaQb;
+        this.deltaQc = deltaQc;
+    }
+
+    @Override
+    public double getDeltaPa() {
```

Comment:
`DeltaPa` is used to increasing modelling of load `P0` for load with a phase asymmetry.  Using that attribute makes `P0` the positive sequence. `DeltaPa` models the unbalanced active power on phase A.
What you do think?

Reply:
DeltaPa is used to extend the modelling of load P0 for load with a phase asymmetry. Using that attribute makes P0 the balanced part of the power of the load. DeltaPa models the additional unbalanced active power part on phase A. Therefore the complete load to be taken into account in calculations for phase A is PA = P0 + DeltaPa